cluster-autoscaler/main.go

```diff
@@ -226,12 +227,13 @@ func main() {
 
 	go func() {
 		http.Handle("/metrics", prometheus.Handler())
+		http.Handle("/health-check", healthCheck)
 		err := http.ListenAndServe(*address, nil)
 		glog.Fatalf("Failed to start metrics: %v", err)
 	}()
 
 	if !leaderElection.LeaderElect {
-		run(nil)
+		run(healthCheck)
```

Comment:
It looks to me like we're not calling StartMonitoring() at all in this codepath?